### PR TITLE
#56 double click changes account

### PR DESCRIPTION
**Pull request by mohsinshabir1234**

### CODE REVIEW

File: gui/control.js [hunk 1]
```diff
@@ -45,12 +54,13 @@ function render() {
...
-            if (!error) return;
-            showToast("Error!", errorToMessage[error])
-        });
+        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', ()=>{
+          switchAccountHandler(nickname);
+      }); 
+
+      document.getElementById(`nick-${nickname}`).addEventListener('dblclick', ()=> {
+          switchAccountHandler(nickname);
+      });
```
Comment (adsau59) [inline gui/control.js:63]: if you are using arrow function you dont really need to use curly braces
```
document.getElementById(`switch-btn-${nickname}`).addEventListener('click', ()=>switchAccountHandler(nickname););

 document.getElementById(`nick-${nickname}`).addEventListener('dblclick', ()=>switchAccountHandler(nickname));
```
should work

**Review by adsau59 — CHANGES_REQUESTED** [review]
just one change requested, everything else looks good

File: gui/control.js (hunk 1 above)
Comment (mohsinshabir1234) [reply gui/control.js]: Done

**Review by mohsinshabir1234 — COMMENTED** [review]
Done with the change

**Pull request merged**